### DISCUSSION TITLE
Run without docker

### DISCUSSION THREAD

**Discussion by MoldyTaint** [SquirrelCorporation/SquirrelServersManager#409] [Ideas]
I genuinely use docker on my VMs/LXC's for Proxmox rather than ever on my PC but feel odd using it on the systems that it would be monitoring if that makes sense.

**Comment by SquirrelDeveloper** [comment]
Hi @MoldyTaint 
I am working on dckerless install with Proxmox VM :-)

**Comment by MoldyTaint** [comment]
Hey that's awesome! Admittedly I tried using some AI to help me get a github workflow that created an arch repo to use but it didn't go as well as I was hoping for. Thank you for the consideration. I am looking forward to using it a bit!

**Comment by SquirrelDeveloper** [comment]
Hi @MoldyTaint .
Could you test on your install this install script for proxmox ?
```
bash -c "$(wget -qLO - https://github.com/SquirrelDeveloper/Proxmox/raw/main/ct/squirrelserversmanager.sh)"
```

**Reply by MoldyTaint** [reply]
Hi @SquirrelDeveloper, Thanks for getting back so soon! Is this to be ran directly on proxmox or on another computer as well? 

I'm looking for a solution that's 1000% what Squirrel Manager does but without the "Main" application being ran on the servers I am actually managing. Being able to load up Squirrel Manager on my desktop or laptop and manage the cluster/vms/lxc rather than going to an IP on the server and then managing it. Hopefully that cleared up a bit. I'll give this a shot when I get home from work otherwise. Thank you for the handwork and determination!

**Reply by SquirrelDeveloper** [reply]
Hi!
I am not sure that's where I am going, but you can perfectly run on it on your laptop, and then add the proxmox host and its VMs to SSM

**Reply by MoldyTaint** [reply]
Hello, sorry for such a late reply!

While I'm not trying to be needy I'm also trying to figure out and learn in the process as this is all new to me but am looking to gain knowledge. I tried running the script on the laptop but it did not run as it said no manager installation was found. My entire thought process is I just don't want to run docker. Not a fan of docker. Was hoping to install this on my main PC or laptop as a standalone application or even how it is now just without...docker. I use docker engine in an LXC for media management etc but am not wanting to put server managing software on the server it's managing - if that makes sense.

**Reply by SquirrelDeveloper** [reply]
I got you. It is possible, but it is more complicated. Here is a guide to do it: https://squirrelserversmanager.io/docs/install/dockerless
I only support Alpine Linux so I dont disperse myself

**Reply by MoldyTaint** [reply]
Hello, I apologize for such a late reply. I really appreciate your communication and generosity to even entertain such an idea. I took a look at that link you have and I greatly appreciate it! I was not aware it was using a reverse proxy to generate the "GUI". That is probably what threw a little kink into what I was trying to do. Thank you for the insights and update.

With your permission could I download the git and try to see if I can swap out the reverse proxy for an GUI based like QT?

**Reply by SquirrelDeveloper** [reply]
Oh yes no problem
